src/mscorlib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -322,6 +325,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\CompilerServices\ReferenceAssemblyAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\CompilerServices\RuntimeCompatibilityAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\CompilerServices\RuntimeFeature.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\CompilerServices\RuntimeWrappedException.cs" />
```

Comment:
- The CoreRT version of RuntimeWrappedException.cs has proper formatting (no `m_` prefix, fields first). Can we keep the formatting from the CoreRT copy.
- The CoreRT version has one private constructor public to workaround private reflection issues. This needs to be done under ifdef in the shared copy like in other places. It would break .NET Native tests otherwise.

Reply:
Turns out RWE needs it:
`Compiler optimization assumption invalid: EE expects field to exist: System.Runtime.CompilerServices.RuntimeWrappedException:m_wrappedException`

Reply:
It should be trivial to rename on the EE side (src\vm\mscorlib.h).